components/Dialogs/examples/DialogsTypicalUseViewController.m

```diff
@@ -72,6 +69,19 @@ - (void)viewDidLoad {
   ]];
 }
 
+- (MDCContainerScheme *)containerScheme {
+  MDCContainerScheme *scheme = [[MDCContainerScheme alloc] init];
+  scheme.colorScheme = self.colorScheme;
+  scheme.shapeScheme =
+      [[MDCShapeScheme alloc] initWithDefaults:MDCShapeSchemeDefaultsMaterial201809];
+  scheme.typographyScheme =
+      [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201804];
+  if (!_containerScheme) {
```

Comment:
if _containerScheme is not nil, than why create a new MDCCOntainerScheme each time anyway?